client/web/compose/src/components/ModuleFields/Editor/File.vue

```diff
@@ -127,6 +212,92 @@ export default {
         this.value = attachmentID
       }
     },
+    async initializeWebcam () {
+      await this.$nextTick()
+
+      this.video = this.$refs.video
```

Comment:
Do we really need do redefine all these variables?
Can't you just use the this.$refs for the ones you need?